src/components/Observations/map/ObservationMapView.tsx

```diff
@@ -0,0 +1,122 @@
+import { ObservationResults } from 'src/types/Observations';
+import { Box, useTheme } from '@mui/material';
+import MapLegend from 'src/components/common/MapLegend';
+import { PlantingSiteMap } from 'src/components/Map';
+import React, { useEffect, useMemo, useState } from 'react';
+import { MapObject, MapSourceBaseData } from 'src/types/Map';
+import { MapService } from 'src/services';
+import MapLayerSelect, { MapLayer } from 'src/components/common/MapLayerSelect';
+import strings from 'src/strings';
+import MapDateSelect from 'src/components/common/MapDateSelect';
+import { getRgbaFromHex } from 'src/utils/color';
+
+type ObservationMapViewProps = {
+  observationsResults?: ObservationResults[];
+};
+
+export default function ObservationMapView({ observationsResults }: ObservationMapViewProps): JSX.Element {
+  const theme = useTheme();
+
+  const observationsDates = useMemo(() => {
+    return observationsResults
+      ?.map((obs) => obs.completedTime)
+      ?.filter((time) => time)
+      ?.map((time) => time!)
+      ?.sort((a, b) => (Date.parse(a!) > Date.parse(b!) ? 1 : -1));
+  }, [observationsResults]);
+
+  const [selectedObservationDate, setSelectedObservationDate] = useState<string | undefined>();
+  useEffect(() => {
+    if (observationsDates) {
+      setSelectedObservationDate(observationsDates[observationsDates.length - 1]);
+    } else {
+      setSelectedObservationDate('');
+    }
+  }, [observationsDates]);
+
+  const mapData: Record<MapObject, MapSourceBaseData | undefined> = useMemo(() => {
+    if (!selectedObservationDate) {
+      return {
+        site: undefined,
+        zone: undefined,
+        subzone: undefined,
+        permanentPlot: undefined,
+        temporaryPlot: undefined,
+      };
+    }
+
+    const selectedObservation = observationsResults?.find((obs) => obs.completedTime === selectedObservationDate)!;
```

Comment:
can there be more than one?

Reply:
Hmm, I suppose there technically could be if more than one observation were completed on the same day. Not a completely implausible scenario

Reply:
I don't think the designs account for this situation either - maybe it's only theoretically possible. We could leave things as-is for now. And bring it up in any of our meetings as a question.

Reply:
I'll go ahead and add a question to the figma

Reply:
Thanks Charlie!! Saw the comment 👍 and helped me understand the situation a bit more. 
The observation's completion date is the latest date a monitoring plot had an observation submitted - at any point in time. Looks like the chances of last updated monitoring plots having same month/year completion dates across planting sites is likely (ofc we can wait for the PRD response). 
Not sure design needs to capture anything here. Like you mentioned, we make updates to the slider (to dedupe by month/year).. and the map renders data for all observations matching a selected completion date (month/year). Rendering multiple observations in the map is already a requirement anyways. map slider probably has the most changes for sure.

Reply:
Basically.. completion time for an observation will change over time as monitoring plots are observed. It will probably stay frozen once all involved planting subzones are `finished planting` (some nuances around undoing `finished planting` but not relevant here).

Reply:
>Looks like the chances of last updated monitoring plots having same month/year completion dates across planting sites is likely

That's true, but the map is only going to show a single planting site at a time, so for a single planting site this will be unlikely.

>It will probably stay frozen once all involved planting subzones have fully observed monitoring plots.

Since the selector we're basing the observationResults on is only selecting completed observations, we're probably okay on this front

Reply:
Ah yes, good point on the map showing a single planting site (completely missed factoring that in) - perhaps just one running observation at a time. Thanks 🙏 
Probably not relevant anymore but just addressing the last comment. The completed observations selector shows observations with a 'completed time' (since upcoming observations or overdue observations will not have one and we don't want to show those). Completed observations here doesn't imply fully completed.

Reply:
Good to know. Thank you